classes/01class/exercises/c01-git03/mfreitassm/README.md

```diff
@@ -0,0 +1,25 @@
+# c01-git03
+
+## Questions (answer inline)
+
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+
+Yes, before added the big file (100M) repo size was 168k and after added was 201M.
+
+2. What is the reason for this problem?
+
+A file copy was stored on the folder /.git/objects as history.
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+
+Git history will need to be rewrited.
+git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch path_to_file" HEAD 
+git push -all
```

Comment:
if the commits were rewritten, here you need to `--force` the push

Reply:
Okay!
Thanks!